tensorflow/core/kernels/mkl_conv_ops.cc

```diff
@@ -271,21 +350,56 @@ class MklConvFwdPrimitive : public MklPrimitive {
       context_.fwd_pd.reset(new ConvFwdPd(*context_.fwd_desc, cpu_engine_));
     }
 
+#ifndef ENABLE_MKLDNN_V1
     // Store the expected memory format
     context_.src_fmt = static_cast<mkldnn::memory::format>(
         context_.fwd_pd.get()->src_primitive_desc().desc().data.format);
 
     context_.filter_fmt = static_cast<mkldnn::memory::format>(
         context_.fwd_pd.get()->weights_primitive_desc().desc().data.format);
+#endif
 
+#ifdef ENABLE_MKLDNN_V1
     // Create memory primitive based on dummy data
+    context_.src_mem.reset(
+        new memory(context_.fwd_pd.get()->src_desc(), cpu_engine_, DummyData));
```

Comment:
These could be defined at the beginning too.
`src_desc` vs `src_primitive_desc`
`weights_desc` vs `weights_primitive_desc`
`dst_desc` vs `desc_primitive_desc`

Reply:
Done.